src/graphql/queries/user.graphql

```diff
@@ -1,4 +1,4 @@
-query GetCurrentUser($address: ID!) {
+query GetUserByAddress($address: ID!) {
```

Comment:
Renamed as it better reflects what the query actually does. The same change is already in #257